qiskit/synthesis/discrete_basis/solovay_kitaev.py

```diff
@@ -75,33 +146,44 @@ def load_basic_approximations(data: list | str | dict) -> list[GateSequence]:
         if isinstance(data, list):
             return data
 
-        # if a file, load the dictionary
+        # file is ``.npy``, load the dictionary it contains
```

Comment:
here it's still npy?

Reply:
This handles both `.npy` and the new bincode format -- I added a comment to make this more clear